README.md

```diff
@@ -950,6 +950,24 @@ This section provides a description of metrics related to MQGET, MQCB and MQCTL
 <td>Shows the number of calls to MQCTL.</td>
 <td>MQCTL count</td>
 </tr>
+<tr>
+<td>mqobject_get_average_destructive_mqget_persistent_message_size_bytes</td>
+<td>gauge</td>
+<td>Shows a average amount of bytes per persistent message that are returned to MQGET.</td>
```

Comment:
Shows **an** average amount of bytes per persistent message that are returned **by** MQGET. 
(and all descriptions below have same typos)